Apps/Admin/Client/Pages/AgentAccessPage.razor

```diff
@@ -110,6 +110,17 @@
         </PagerContent>
     </MudTable>
 
+    @if (Agents.Count() == SearchResultLimit)
```

Comment:
Isn’t it typical for warning messages to appear at the top of the screen?

Reply:
The alert was positioned according to the mockup.